pueblo/testing/folder.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+
+def list_files(path: Path, pattern: str):
```

Comment:
minor: This method seems to not be tested separately, (of course it's used by the other 2 which are tested), maybe add a test for this one as well?

Reply:
For now, I think will be good enough. I don't want to over-engineer the code here. There are lots of opportunities for making the tests even stronger, currently there are gaps, specifically on evaluating exception handling. Maybe I will spend some extra minutes on it, on behalf of a different iteration. Right now, I would like to move on.

Reply:
thank you!